<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkDetailManager</class>
 <widget class="QDialog" name="WorkDetailManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Work Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="presets_label">
         <property name="text">
          <string>Presets:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="WorkDetailPresetView" name="presets_view">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="workdetails_label">
         <property name="text">
          <string>Current Work details:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="WorkDetailView" name="workdetails_view">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QToolButton" name="move_top_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Move to top</string>
             </property>
             <property name="icon">
              <iconset theme="go-top"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="move_up_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Move up</string>
             </property>
             <property name="icon">
              <iconset theme="go-up"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="move_down_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Move down</string>
             </property>
             <property name="icon">
              <iconset theme="go-down"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="move_bottom_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Move to bottom</string>
             </property>
             <property name="icon">
              <iconset theme="go-bottom"/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="remove_workdetails_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove selected</string>
             </property>
             <property name="icon">
              <iconset theme="edit-delete"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="add_workdetail_button">
             <property name="text">
              <string>Add new work detail</string>
             </property>
             <property name="icon">
              <iconset theme="document-new"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorkDetailView</class>
   <extends>QListView</extends>
   <header>WorkDetailManager.h</header>
  </customwidget>
  <customwidget>
   <class>WorkDetailPresetView</class>
   <extends>QTreeView</extends>
   <header>WorkDetailManager.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>WorkDetailManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>WorkDetailManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
